### PR TITLE
Implement viewing of meetings

### LINKED ISSUE #10: As a user, I want to view all meetings that I have

### PR DESCRIPTION
**Pull request by chuakid**
Resolves #10

### CODE REVIEW

**Comment by jeremykhoo-NUS** [comment]
LGTM

File: src/main/java/seedu/address/logic/commands/ViewMeetingsCommand.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package seedu.address.logic.commands;
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.meeting.Meeting;
+
+
```
Comment (Beebeeoii) [inline src/main/java/seedu/address/logic/commands/ViewMeetingsCommand.java:8]: could we remove extra lines to standardise?

File: src/main/java/seedu/address/logic/commands/ViewMeetingsCommand.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package seedu.address.logic.commands;
```
Comment (Beebeeoii) [inline src/main/java/seedu/address/logic/commands/ViewMeetingsCommand.java:1]: could we have an empty line after the package before other imports to standardise

**Review by Beebeeoii — CHANGES_REQUESTED** [review]
lgtm other than some minor nits ! thanks !

**Review by Beebeeoii — APPROVED** [review]

**Pull request merged**